guides/common/modules/ref_port-and-firewall-requirements.adoc

```diff
@@ -1,4 +1,4 @@
-[id="Ports_and_Firewalls_Requirements_{context}"]
+[id="Port_and_firewall_requirements_{context}"]
```

Comment:
Just a comment. It would be good to have a consistent format for anchor IDs.

Reply:
Yeah. Added:  https://community.theforeman.org/t/documentation-team-meeting-2024-02-08/36733?u=maximilian